app.js

```diff
@@ -1,10 +1,28 @@
+// Get dependencies
 var express = require('express');
 var path = require('path');
 var favicon = require('serve-favicon');
```

Comment:
is this even needed? you aren't using `favicon` anywhere (and similarly to many of the other declared variables)